src/app/ng-intercom/intercom/intercom.ts

```diff
@@ -248,6 +237,8 @@ export class Intercom {
     //   return
     // }
 
+    this.config = config
```

Comment:
Why is this necessary?

Reply:
Because the config (app_id) is later used in the `boot` function.